src/frontend/apps/drive/src/utils/itemUtils.tsx

```diff
@@ -0,0 +1,8 @@
+import { Item, ItemType } from "@/features/drivers/types";
+
+export const itemIsWorkspace = (item: Item) => {
```

Comment:
imo we should try to fit it inside `features`, as the item types are in `features/drivers`, we could create a directory named `features/drivers/utils`? WDYT?

Having it not attached to a feature that already define items feels weird imo